generators/java/okhttp-gson/templates/FileTransfer.mustache

```diff
@@ -0,0 +1,96 @@
+{{>licenseInfo}}
```

Comment:
Not sure if we should include the license block in this file, since it's not part of OpenAPI generator 🤔